sonic_api.yaml

```diff
@@ -1400,3 +1604,24 @@ definitions:
       ip_addr:
         type: string
         description: IP address of directly attached device to the switch
+  TunnelEntry:
+    type: object
+    properties:
+      tunnel_name:
+        type: string
+      description:
+        type: string
+      src_ip:
+        type: string
+        format: ipv4
```

Comment:
Why only ipv4?

Reply:
We will add ipv6 support in later iterations.